styles.css

```diff
@@ -294,7 +294,7 @@ button:active {
 
 .card li {
   font-size: 0.8em;
-  color: #36b37f;
+  color: #227150;
```

Comment:
A handful of small boxes' text was hard to read with the light green. Instead of changing the box's color, we changed the text's color